.cirrus.yml

```diff
@@ -198,7 +198,7 @@ task:
   libs_cache:
     folder: build/libs
     fingerprint_script: echo "`md5sum lib/CMakeLists.txt` cross arm 20200423"
-    populate_script: make libs build_flags=-j8 llvm_archs="X86;ARM"
```

Comment:
what happened to the arch change? is that not needed anymore?

Reply:
Rather than removing it, maybe we need to update to `llvm_archs="ARM;X86;AArch64"`?

Reply:
I made `X86;ARM` the default in `lib/CMakeLists.txt`.

Reply:
@jemc Should we add `AArch64` to `lib/CMakeLists.txt` as well?

Reply:
Yes, `AArch64` is needed too

Reply:
Added.